.vscode/settings.json

```diff
@@ -20,5 +24,8 @@
       "url": "./.vscode/package.json-schema.json"
     }
   ],
+  "tailwindCSS.classFunctions": ["classnames", "classNames"],
+  "tailwindCSS.validate": true,
+  "tailwindCSS.lint.cssConflict": "error",
```

Comment:
`tailwindCSS.validate` turns on rules for validation. We are only using one `tailwindCSS.lint.cssConflict` which catches conflicting classnames. The tailwind eslint plugin will also be added to catch these errors during `yarn lint`

![Screenshot 2025-07-09 at 1 37 57 PM](https://github.com/user-attachments/assets/e46661cb-2eda-46aa-ab10-f7984f9fbde1)